src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -2303,7 +2324,45 @@ private void visitRecordType(ObjectType type) {
       emit("}");
     }
 
-    private Set<String> getSortedPropertyNamesToEmit(final ObjectType type) {
+    /**
+     * Returns the (sorted) set of properties that should be emitted within types, i.e. on classes,
+     * interfaces, or in object types.
+     *
+     * @param isStatic if true, skips properties that are likely namespaces during extern
```

Comment:
this is a bit odd, but matches previous behaviour, I only moved the code around. I guess we had a good reason for that?

Reply:
comment on line 2337 and extern_static_namespace.externs.js example. That said, if I commented out that line all tests pass, so maybe we fixed it some other way.

Externs is significant because "namespaces' in extersn are declared without a 'goog.provide'.